Chart.yaml

```diff
@@ -9,5 +9,5 @@ keywords:
   - airflow
 dependencies:
   - name: airflow
-    version: 1.11.0
+    version: 1.13.1
```

Comment:
Should we update line 4 too?

Reply:
i have made a all the references properly i dont see anything missing

Reply:
Oh okay now i got it